src/js/components/CosmosErrorMessage.js

```diff
@@ -43,7 +43,17 @@ class CosmosErrorMessage extends React.Component {
       );
     }
 
-    return <Trans id={message} render="span" />;
+    if (message) {
+      return <Trans id={message} render="span" />;
+    }
+
+    i18nMark("Looks Like Something is Wrong. Please try again.");
```

Comment:
We don't need this with the Trans component on line 53, your "marking" this string twice essentially.